<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addInTreeDialog</class>
 <widget class="QDialog" name="addInTreeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="passportNum_label">
           <property name="text">
            <string>Номер паспорта</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="placeAndDate_label">
           <property name="text">
            <string>Место и дата выдачи паспорта</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="name_label">
           <property name="text">
            <string>ФИО</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="birthDate_label">
           <property name="text">
            <string>Год рождения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="adress_label">
           <property name="text">
            <string>Адрес</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="passportNumEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="placeAndDateEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="nameEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="birthDateEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="adressEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
